shoop/core/models/_contacts.py

```diff
@@ -92,9 +96,25 @@ def __init__(self, *args, **kwargs):
             kwargs.setdefault("tax_group", self.default_tax_group_getter())
         super(Contact, self).__init__(*args, **kwargs)
 
+    def save(self, *args, **kwargs):
+        super(Contact, self).save(*args, **kwargs)
+        if self.default_contact_group_identifier:
```

Comment:
This will happen on every save, not just the first one?
So one can't ever remove a contact from their default contact group?